### PR TITLE
Feature/my values

### PR DESCRIPTION
**Pull request by FaysalSolutions**
What the title says

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.7.20'
+    ext.kotlin_version = '1.8.0'
```
Comment (mustiwebp) [inline android/build.gradle:2]: Just because of curiosity, why do we need this?

File: lib/api/backend_values_api.dart [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import 'package:flutter/foundation.dart';
+import 'package:parse_server_sdk/parse_server_sdk.dart';
+import 'package:picos/models/values.dart';
+
+import '../models/daily.dart';
+import '../models/weekly.dart';
+
+/// API for storing values at the backend.
+class BackendValuesApi {
+  /// Ret
```
Comment (mustiwebp) [inline lib/api/backend_values_api.dart:27]: Incomplete comment

File: lib/models/values.dart [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+///
+class Values {
+  /// Creates a MyValues object.
+  Values({
+    required this.dailyList,
+    required this.weeklyList,
+  });
+
+  ///
```
Comment (mustiwebp) [inline lib/models/values.dart:29]: Empty comment

File: lib/models/values.dart [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  List<Daily>? dailyList;
+
+  ///
```
Comment (mustiwebp) [inline lib/models/values.dart:32]: Empty comment

File: lib/screens/home_screen/overview/widgets/contact_section.dart [hunk 5]
```diff
@@ -23,7 +23,7 @@ import 'package:picos/themes/global_theme.dart';
 
 /// Widget which shows a graph
 class ContactSection extends StatelessWidget {
```
Comment (mustiwebp) [inline lib/screens/home_screen/overview/widgets/contact_section.dart:25]: Please rename 'ContactSection' or change comment

File: lib/widgets/picos_select.dart [hunk 6]
```diff
@@ -107,12 +107,14 @@ class _PicosSelectState extends State<PicosSelect> {
         absorbing: widget.disabled,
         child: ButtonTheme(
           alignedDropdown: true,
+
```
Comment (mustiwebp) [inline lib/widgets/picos_select.dart:110]: Why is an empty line here?

File: lib/widgets/picos_chart_column.dart [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+          ColumnSeries<PicosChartSampleData, String>(
+            dataSource: _prepareChartData(),
+            xValueMapper: (PicosChartSampleData point, _) => point.x,
+            yValueMapper: (PicosChartSampleData point, _) => point.y,
+            name: title,
+            dataLabelSettings: const DataLabelSettings(
+              isVisible: true,
+              labelAlignment: ChartDataLabelAlignment.outer,
+            ),
+          )
```
Comment (mustiwebp) [inline lib/widgets/picos_chart_column.dart:149]: Can you put a trailing comma

File: lib/widgets/picos_chart_line.dart [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+            name: title,
+            markerSettings: const MarkerSettings(isVisible: true),
+            dataLabelSettings: const DataLabelSettings(
+              isVisible: true,
+              labelAlignment: ChartDataLabelAlignment.top,
+            ),
+            emptyPointSettings: EmptyPointSettings(
+              mode: EmptyPointMode.drop,
+            ),
+          )
```
Comment (mustiwebp) [inline lib/widgets/picos_chart_line.dart:122]: Can you put a trailing comma here?

File: lib/widgets/picos_select.dart (hunk 6 above)
Comment (FaysalSolutions) [reply lib/widgets/picos_select.dart:110]: This does not belong to my changed files, but no problem I can change that ;)

File: lib/widgets/picos_chart_column.dart (hunk 7 above)
Comment (FaysalSolutions) [reply lib/widgets/picos_chart_column.dart:149]: My ide doesn't show me any warnings about this, but if it helps, I'll do it.

File: lib/widgets/picos_chart_line.dart (hunk 8 above)
Comment (FaysalSolutions) [reply lib/widgets/picos_chart_line.dart:122]: My ide doesn't show me any warnings about this, but if it helps, I'll do it.

File: lib/models/values.dart (hunk 4 above)
Comment (FaysalSolutions) [reply lib/models/values.dart]: resolved in [refactoring (Add comments)](https://github.com/healthcare-it-solutions/picos/pull/184/commits/26d5c6e60feb8cb2eee7b9477f1a1b5f7db36249)

File: lib/models/values.dart (hunk 3 above)
Comment (FaysalSolutions) [reply lib/models/values.dart]: resolved in [refactoring (Add comments)](https://github.com/healthcare-it-solutions/picos/pull/184/commits/26d5c6e60feb8cb2eee7b9477f1a1b5f7db36249)

File: lib/api/backend_values_api.dart (hunk 2 above)
Comment (FaysalSolutions) [reply lib/api/backend_values_api.dart]: resolved in [refactoring (Add comments)](https://github.com/healthcare-it-solutions/picos/pull/184/commits/26d5c6e60feb8cb2eee7b9477f1a1b5f7db36249)

File: lib/screens/home_screen/overview/widgets/contact_section.dart (hunk 5 above)
Comment (FaysalSolutions) [reply lib/screens/home_screen/overview/widgets/contact_section.dart:25]: resolved in [refactoring (Add comments)](https://github.com/healthcare-it-solutions/picos/pull/184/commits/26d5c6e60feb8cb2eee7b9477f1a1b5f7db36249)

File: lib/widgets/picos_chart_column.dart (hunk 7 above)
Comment (FaysalSolutions) [reply lib/widgets/picos_chart_column.dart]: resolved in [refactoring (trailing comma)](https://github.com/healthcare-it-solutions/picos/pull/184/commits/51cfaea476814bf6d057b648b4136a886434d7de)

File: lib/widgets/picos_chart_line.dart (hunk 8 above)
Comment (FaysalSolutions) [reply lib/widgets/picos_chart_line.dart]: resolved in [refactoring (trailing comma)](https://github.com/healthcare-it-solutions/picos/pull/184/commits/3e2a7a2a2fe5b0b388ecb61ec59facc80f4a5449)

**Review by mustiwebp — APPROVED** [review]

**Pull request merged**